examples/simple-authenticated-api/lib/simple-authenticated-api-stack.ts

```diff
@@ -154,5 +154,10 @@ export class SimpleAuthenticatedApiStack extends cdk.Stack {
       path: "/api-documentation",
       method: apigatewayv2.HttpMethod.GET,
     });
+
+    console.log(`Regional domain name: ${api.domainName.regionalDomainName}`);
```

Comment:
Could register these as a CFN Output from the stack but this is fine to show they are available